tests/test_nitcorn.nit

```diff
@@ -17,8 +17,8 @@ import pthreads
 
 redef class Sys
 	var iface: String is lazy do
-		srand
-		return "localhost:{10000+20000.rand}"
+		var testid = "NIT_TESTING_ID".environ.to_i
+		return "localhost:{10000+testid}"
```

Comment:
Many edge cases here:
* You may get a port outside the [ephemeral port range](https://en.wikipedia.org/wiki/Ephemeral_port).
* You may collide with a non-Nitcorn process.

You should let the OS chose a port for you.